final_project/app/Http/Controllers/SubscriptionController.php

```diff
@@ -36,8 +36,7 @@ public function create_or_activate(Request $request)
 
     public function deactivate($id) {
         $subscription = Subscription::find($id);
-        $subscription->active = false;
-        $subscription->save();
+		Subscription::destroy($id);
```

Comment:
The reason we had active here was so that a user could unsubscribe then resubscribe to a show and not lose track of the episodes they had watched.